docs/pages/web/modal.js

```diff
@@ -244,6 +245,59 @@ export default function ModalPage({ generatedDocGen }: {| generatedDocGen: DocGe
             }
           />
         </MainSection.Subsection>
+        <MainSection
+          name="Mobile"
+          description={`Modal requires [DeviceTypeProvider](/web/utilities/devicetypeprovider) to enable its mobile user interface. The example below shows the mobile platform UI and its implementation.
+
+For mobile, all \`size\` are unified into a full mobile viewport Modal. Notice that \`subHeading\` gets moved from the header to the main content container.
```

Comment:
typo: "For mobile, all _sizes_ are unified…"

Not sure if "mobile" is necessary the second time. I would think this would be "…into a _full-viewport_ modal."